x-pack/plugins/security_solution/public/timelines/components/side_panel/index.tsx

```diff
@@ -18,14 +17,6 @@ import { EventDetailsPanel } from './event_details';
 import { HostDetailsPanel } from './host_details';
 import { NetworkDetailsPanel } from './network_details';
 
-// TODO: EUI team follow up on complex types and styled-components `styled`
-// https://github.com/elastic/eui/issues/4855
-const StyledEuiFlyout: StyledComponent<typeof EuiFlyout, {}, { children?: ReactNode }> = styled(
```

Comment:
@thompsongl, not sure if this comment still applies here, but I've removed the styled component in favor of a simple `EuiFlyout`

Reply:
If you've removed `StyledEuiFlyout` then the comment no longer applies
Thanks for the ping 👍